server/src/migrations/20210208161100-create-sso.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.createTable('SingleSignOnRequests', {
+            id: {
+                primaryKey: true,
+                allowNull: false,
+                type: Sequelize.UUID,
```

Comment:
```suggestion
                type: Sequelize.STRING,
                defaultValue: Sequelize.UUIDV4
```
hatte da mal ein problem, weil da UUID drin stand. Deshalb überall bitte string. Effektiv machst du ja sowieso BeforeCreate() und vergibst die selbst

Reply:
Stimmt, dann ging WHERE id = nicht mehr.